packages/ui/cypress/support/commands.ts

```diff
@@ -109,6 +110,16 @@ Cypress.Commands.add('connectAccounts', (accountAddresses = [Account1] as string
   })
 })
 
+Cypress.Commands.add('addWatchAccount', (address: string, name?: string) => {
```

Comment:
This should not be used all over because it's slow, manually adding a watch account is already tested extensively. Please use `visitWithLocalStorage` to use watched accounts, if what you want is test another functionality.